src/components/views/elements/ImageView.tsx

```diff
@@ -160,41 +160,82 @@ export default class ImageView extends React.Component<IProps, IState> {
         });
     };
 
-    private zoom(delta: number) {
-        const newZoom = this.state.zoom + delta;
+    private zoomDelta(delta: number, zoomX?: number, zoomY?: number) {
+        this.zoom(this.state.zoom + delta, zoomX, zoomY);
+    }
+
+    private zoom(zoomLevel: number, zoomX?: number, zoomY?: number) {
```

Comment:
Calling these `zoomX` and `zoomY` feels confusing

Reply:
Hmm, perhaps `anchorX`/`anchorY`?

Reply:
Yeah, sounds much better